src/common/iron-session/iron-session-provider.ts

```diff
@@ -0,0 +1,22 @@
+export type SealDataOptions = {
+  password:
+    | string
+    | {
+        [id: string]: string;
+      };
+  ttl?: number | undefined;
+};
+
+export type UnsealedDataType = Record<string, unknown>;
+
+/**
+ * Interface encapsulating the sealData/unsealData methods for separate iron-session implementations.
```

Comment:
Let's add a comment for future generations on why we're doing this, plus that we can effectively deprecate this when we drop support for Node 16.

Reply:
Adding!